util/searchRankScript.bash

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Clone the package to be published from latest commit
+package=$(git diff --name-only HEAD HEAD~1)
+end=".end"
+
+# check for TOML file
+if [[ $package != *".toml"* ]]; then
+  echo "Not TOML FILE"
+  exit 0
+fi
+path="$package$end"
+cd "$(dirname "$path")" || exit 1
```

Comment:
I generally prefer not changing the directory (especially if there are multiple changes) in scripts like these, because it introduces a "state" to our script which can make debugging / reproducing issues more difficult.

I'd suggest prepending paths where in the variables used and staying in the top-level directory if nothing prohibits that.